### ISSUE TITLE
wsl2-ssh-pageant.exe is blocked by Crowdstrike

### ISSUE THREAD

**Issue by Gummibaer83** [BlackReloaded/wsl2-ssh-pageant#40]
this version of wsl2-ssh-pageant.exe is blocked by Crowdstrike for "Machine Learning via Sensor-based ML".
So sadly I can not use it on my work computer.
Can you somehow fix this ?

**Comment by davidshen84** [comment]
- Are you talking about https://www.crowdstrike.com/?
- Can you be more specific about what means *blocked* in this context?

Security software and the companies that use them can apply their own security rule. We cannot advise on that.

If you or your organisation spotted any security concerns in this repository, please let us know.

**Comment by Gummibaer83** [comment]
Yes, my company uses crowdstrike and when i download the latest version it is immediately deleted because it is detected as  "Machine Learning via Sensor-based ML". So there has to be something in this version that triggeres this detection.

**Comment by davidshen84** [comment]
Or, your company should update their definition of what is "Machine Learning via Sensor-based ML".

**Comment by Gummibaer83** [comment]
seems like this is a long known Problem: https://www.reddit.com/r/crowdstrike/comments/gaep5g/detection_and_blcok/
This happens when the EXE File doesn't use "Prevention Hashes"

**Comment by davidshen84** [comment]
If you read through the comments, it says you or your admin needs to
maintain that "Prevention Hashes" in your system.

**Comment by Gummibaer83** [comment]
yeah our admin will not do that for your software, in that reddit case they had people that programmed stuff not 3rd party software.
So seems like there isn't a solution for that.
